modules/ossm-threescale-anatomy-of-the-configuration.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-anatomy-of-the-configuration_{context}"]
+= The anatomy of the 3scale WebAssembly module configuration
+
+The anatomy of the 3scale WebAssembly module configuration depends on the 3scale account and authorization service, and the list of services to handle.
```

Comment:
would architecture be a better word than anatomy?

Reply:
+1